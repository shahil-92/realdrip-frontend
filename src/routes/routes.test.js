import React from 'react';
import Routes from '.';

describe('<Routes />', () => {
  describe('Rendering', () => {
    it('should render correctly', () => {
      const wrapper = shallow(<Routes />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});

