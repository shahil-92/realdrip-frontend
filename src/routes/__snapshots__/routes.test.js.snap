// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes /> Rendering should render correctly 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(Signup)",
          "type": [Function],
        }
      }
      exact={true}
      path="/signup"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/email-confirmation"
    />
  </Switch>
</BrowserRouter>
`;
